.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,153 @@
+            - name : Process pass/fail info
+              id: pass-fail
+              run: |
+                # fail.txt is created when at least one test fails.
+                # job.status can't be used because the subsequent steps are not
+                # executed once a step fails.
+                if [ -e ${{ env.test_results_directory }}/fail.txt ]; then
+                    echo "pass_fail=❌" >> "$GITHUB_OUTPUT"
+                    echo "exit_value=1" >> "$GITHUB_OUTPUT"
+                else
+                    echo "pass_fail=✅" >> "$GITHUB_OUTPUT"
+                    echo "exit_value=0" >> "$GITHUB_OUTPUT"
+                fi
+            - name: Report to Slack
+              id: slack
+              uses: slackapi/slack-github-action@v1.26.0
+              with:
+                payload: |
+                  {
+                    "blocks":[
+                      {
+                        "type": "header",
+                        "text": {
+                          "type": "plain_text",
+                          "text": "${{ steps.pass-fail.outputs.pass_fail }} ${{ env.browser }} :firefox: ${{ env.xcodebuild_test_plan }} - ${{ env.ios_simulator }} iOS ${{ env.ios_version }}",
+                          "emoji": true
+                        }
+                      },
+                      {
+                        "type":"divider"
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                          "type": "mrkdwn",
+                          "text": "*Branch*: `${{ github.ref_name }}`"
+                        },
+                        "fields": [
+                          {
+                            "type": "mrkdwn",
+                            "text": "<https://storage.googleapis.com/mobile-reports/public/${{ env.browser }}/${{ env.xcodebuild_test_plan }}/result_${{ github.run_number }}/index.html|*Test Report*> :debug:"
+                          },
+                          {
+                            "type": "mrkdwn",
+                            "text": "<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|*Github Actions Job*> :github:"
+                          }
+                        ]
+                      },
+                      {
+                        "type":"divider"
+                      },
+                      {
+                        "type": "context",
+                        "elements": [
+                          {
+                            "type": "mrkdwn",
+                            "text": ":testops-notify: created by <https://mozilla-hub.atlassian.net/wiki/spaces/MTE/overview|Mobile Test Engineering>"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+              env:
+                SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+            - name: Report status
```

Comment:
Should we let Github Actions to report the test run as fail if one or more tests did not pass?